src/controllers/CreateFormController.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+const FormValidator = require("../validators/FormValidator");
+const constants = require("../constants");
+
+module.exports = class CreateFormController {
+  constructor(formRepository, resultsRepostitory) {
+    this.path = "/create";
+    this.router = express.Router();
+    this.formRepo = formRepository;
+    this.resultsRepo = resultsRepostitory;
+
+    this.initializeRoutes();
+    console.log("CreateFormController initialized");
+  }
+
+  initializeRoutes() {
+    this.router.route(`${this.path}`).post(this.post);
+  }
+
+  post = async (req, res) => {
+    const form = this.getForm(req.body);
+
+    if (FormValidator.isValid(form)) {
+      const id = this.formRepo.save(form);
```

Comment:
zapytania do właściwej bazy są asynchroniczne, trzeba robić await
`const id = this.formRepo.save(form);`